src/content/self-hosted/administration/configuration.mdx

```diff
@@ -157,7 +157,19 @@ auth:
 
 > You can also use a [secret](self-hosted/administration/configuration.mdx/#cloud-provider-secret) to store the sensitive part of these credentials.
 
-The `organization` field is optional. Only members of the organization will be allowed to log in into your Okteto instance. An empty `organization` field permits any user to log in.
+The `organization` field is optional. Only members of the organization will be allowed to log in into your Okteto instance. An empty `organization` field permits any user to log in. 
+
+If you want to provide access to only select users of your organization, you can additionally use the `allowList` key to list the GitHub accounts you want to allow.
```

Comment:
we should mention how organization and allowList work together. Or maybe we should pitch it as use one or the other?

Reply:
> Or maybe we should pitch it as use one or the other?

Is that what we want? I'm not sure on the intended use here. 

Do you think it's okay if we have these two fields documented in isolation (like right now) and let users connect the dots and decide if they want to use a combination of both or either one.